tests/foreman/cli/test_contentview.py

```diff
@@ -5136,7 +5139,49 @@ class ContentViewFileRepoTestCase(CLITestCase):
     """Specific tests for Content Views with File Repositories containing
     arbitrary files
     """
-    @stubbed()
+    @classmethod
+    def setUpClass(cls):
+        """Create a product and an org which can be re-used in tests."""
+        super(ContentViewFileRepoTestCase, cls).setUpClass()
+        cls.org = make_org()
+        cls.product = make_product({'organization-id': cls.org['id']})
+
+    def _make_file_repository_upload_contents(self, options=None):
+        """Makes a new File repository, Upload File/Multiple Files
+        and asserts its success """
+        if options is None:
+            options = {}
+
+        options_list = (u'product-id', u'content-type', u'url', u'name')
+        if None in [options.get(keys) for keys in options_list]:
+            options[u'product-id'] = self.product['id']
+            options[u'content-type'] = 'file'
+            options[u'url'] = CUSTOM_FILE_REPO
+            options[u'name'] = gen_string('alpha')
+
+        new_repo = make_repository(options)
+        remote_path = "/tmp/{0}".format(RPM_TO_UPLOAD)
+        if 'multi_upload' not in options or not options['multi_upload']:
+            ssh.upload_file(
+                local_file=get_data_file(RPM_TO_UPLOAD),
+                remote_file=remote_path
+            )
+        else:
+            remote_path = "/tmp/{}/".format(gen_string('alpha'))
```

Comment:
here as well

Reply:
@ekohl  @omkarkhatavkar will not recommend to use os.path.join for remote path.